### ISSUE TITLE
Question: missing hibernate-redis-client jar?

### ISSUE THREAD

**Issue by vchau** [debop/hibernate-redis#55]
Hi,
I'm trying to use the 2.0.2 release and have added the maven dependencies as explained in the readme but maven is complaining that `Could not find artifact com.github.debop:hibernate-redis-client:jar:2.0.2`

This is what I added in my pom.xml
```
<dependency>
    <groupId>com.github.debop</groupId>
    <artifactId>hibernate-redis</artifactId>
    <version>2.0.2</version>
</dependency>
```

I've also added the jcenter repository into my pom and am able to see the other jars.  When I browse to the jcenter repo, I can see the hibernate-redis and hibernate-redis-parent artifacts, but no hibernate-redis-client artifact.  Am I missing something?

**Comment by debop** [comment]
I missing shadowJar tasks (merge sub module jar files)
I published new version 2.0.3 
please using 2.0.3

sorry for inconvience for using^^

**Comment by vchau** [comment]
Worked. Thank you.
